pulp_container/app/content.py

```diff
@@ -6,9 +6,19 @@
 registry = Registry()
 
 app.add_routes(
-    [web.get(r"/pulp/container/{path:.+}/blobs/sha256:{digest:.+}", registry.get_by_digest,)]
+    [
+        web.get(
+            r"/pulp/container/{path:.+}/blobs/sha256:{digest:.+}",
+            registry.get_by_digest,
+        )
+    ]
```

Comment:
I think, by removing the last comma, we can keep it a one-liner. 
The question is, which one is more readable?